src/components/Navigation/index.js

```diff
@@ -52,6 +55,36 @@ const NavigationAuth = ({ authUser }) => (
       <SignOutButton />
     </li>
```

Comment:
Add new thing in the old Navigation without bootstrap.
Comment out the Bootstrap version.